deploy/olm-catalog/mobile-developer-console-operator/0.2.6/mobile-developer-console-operator.v0.2.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,236 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"mdc.aerogear.org/v1alpha1","kind":"MobileDeveloperConsole","metadata":{"name":"example-mdc"},"spec":{"oAuthClientId":"mobile-developer-console","oAuthClientSecret":"foobar"}}]'
+    capabilities: Basic Install
+  name: mobile-developer-console-operator.v0.2.6
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Placeholder description
+      displayName: Mobile Developer Console Operator
+      kind: MobileDeveloperConsole
+      name: mobiledeveloperconsoles.mdc.aerogear.org
+      version: v1alpha1
+  displayName: Mobile Developer Console Operator
+  install:
+    spec:
+      deployments:
+      - name: mobile-developer-console-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: mobile-developer-console-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: mobile-developer-console-operator
+            spec:
+              containers:
+              - command:
+                - mobile-developer-console-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: mobile-developer-console-operator
+                - name: OPENSHIFT_HOST
+                  value: ${OPENSHIFT_HOST}
+                - name: UPS_DOCUMENTATION_URL
+                - name: IDM_DOCUMENTATION_URL
+                - name: SYNC_DOCUMENTATION_URL
+                - name: MSS_DOCUMENTATION_URL
+                image: quay.io/aerogear/mobile-developer-console-operator:ae712e3
```

Comment:
I think this should be a semver and not a git hash?

Reply:
this is also another reason I think we should bump the version to include the mobile client admin changes, that is what this label is for.

Reply:
Let's leave this as-is for now. We'll be doing a new release in the next couple of days anyway, so if we can get this PR merged before then, we can create the new version of the CSV as part of the release.

Reply:
one thing to notice is that we will need to add the new CSV file in the manifest repo and update the MDC operator version on the MDC integreatly, so it will require a couple PR's to be merged

Reply:
That sounds like something that integreatly automation should handle, and not something that we need to be concerned about here in the mobile-developer-console-operator project, right?